.buildkite/scripts/steps/integration_tests.sh

```diff
@@ -19,7 +19,7 @@ fi
 
 # Run integration tests
 set +e
-AGENT_STACK_VERSION="${STACK_VERSION}" TEST_INTEG_CLEAN_ON_EXIT=true  STACK_PROVISIONER="$STACK_PROVISIONER" SNAPSHOT=true mage $MAGE_TARGET $MAGE_SUBTARGET
+AGENT_VERSION="8.16.0-SNAPSHOT" AGENT_STACK_VERSION="${STACK_VERSION}" TEST_INTEG_CLEAN_ON_EXIT=true  STACK_PROVISIONER="$STACK_PROVISIONER" SNAPSHOT=true mage $MAGE_TARGET $MAGE_SUBTARGET
```

Comment:
```suggestion
AGENT_VERSION="8.16.0-SNAPSHOT" AGENT_STACK_VERSION="8.16.0" TEST_INTEG_CLEAN_ON_EXIT=true  STACK_PROVISIONER="$STACK_PROVISIONER" SNAPSHOT=true mage $MAGE_TARGET $MAGE_SUBTARGET
```

Reply:
This is slightly brutal but it should nullify the 8.17.0 version we read from package.version